model/src/channel/mod.rs

```diff
@@ -145,10 +145,10 @@ pub struct Channel {
     pub permission_overwrites: Option<Vec<PermissionOverwrite>>,
     /// Sorting position of the channel.
     #[serde(skip_serializing_if = "Option::is_none")]
-    pub position: Option<i64>,
+    pub position: Option<i16>,
     /// Amount of seconds a user has to wait before sending another message.
     #[serde(skip_serializing_if = "Option::is_none")]
-    pub rate_limit_per_user: Option<u64>,
+    pub rate_limit_per_user: Option<u32>,
```

Comment:
yes, but I don't want to accidentally break deserialization due to too small integers. If this type truly is stable, like `user_limit` then it would be fine.

Would like a second opinion

Reply:
IMO we can go with u16 here. I don't see a ratelimit of a day happening